.circleci/config.yml

```diff
@@ -117,17 +124,37 @@ build_images: &build_images
     - checkout
     - <<: *exports
     - run: |
+        mkdir -p images/
         read -ra IMG_ARRAY <<< "$IMAGES"
         if [[ -n "${CIRCLE_TAG}" ]]; then
           makeArgs="VERSION=${CIRCLE_TAG}"
         fi
         for IMAGE in "${IMG_ARRAY[@]}"; do
-          make IMG_MODIFIER="$IMG_MODIFIER" IMAGE_TAG="${DEV_TAG}" $makeArgs ${IMAGE}
+          make IMG_MODIFIER="$IMG_MODIFIER" IMAGE_TAG="${DEV_TAG}" $makeArgs kubeapps/${IMAGE}
```

Comment:
Note: I have removed `kubeapps/` from `${IMAGE}` to be able store `${IMAGE}` as a file not `folder/file`. Let me know if that is readable or you prefer to keep using the previous definition of `${IMAGE}`

Reply:
The only thing that might be an issue is if we want to push these images somewhere other than Docker Hub in the future, or need to use different repos for different images, but I don't think that's the case right now.

Reply:
Yes, we can change this later if needed.